.github/workflows/aks-deploy.yaml

```diff
@@ -301,48 +307,52 @@ jobs:
           docker build ${{ inputs.dockerFilePath }} ${{ steps.getsecrets.outputs.buildArguments }} -t "${{ env.containerRegistry  }}/${{ inputs.dockerImageName }}:${{ inputs.dockerImageTag }}"
           docker push "${{ env.containerRegistry }}/${{ inputs.dockerImageName }}:${{ inputs.dockerImageTag }}"
 
-      - name: Create valid release-name
-        id: generate-release-name
-        run: |
-          release=${{ env.appName }}-${{ github.sha }}
-          release=${release::53}
-          release=$(echo ${release//[!0-9a-zA-Z]/-} | tr '[:upper:]' '[:lower:]' | sed -e 's/^-/z/' -e 's/-$/z/')
-          echo "release=$release" >> $GITHUB_ENV
-
       - name: Create values override file
         run: |
           WEB_AUTHENTICATION="${{ inputs.webAuthentication }}"
-          ADMIN_INGRESS_WHITELIST="${{ inputs.adminIngressWhitelist }}"
-          INGRESS_WHITELIST="${{ inputs.ingressWhitelist }}"
-          if [[ ${WEB_AUTHENTICATION} == "true" ]] ; then
+          INGRESS_WHITELIST_ENV="${{ env.ingressWhitelist }}"
```

Comment:
What type of info is in the INGRESS_WHITELIST?  Is this like IP addresses internally?

Reply:
This new variable is to allow for transitioning away from and consolidating the multiple Helm environment directories in repositories. This is one of the few Helm values that are set on a per-environment basis. 

Setting it in the reusable workflow/inputs here allows us to change the "whitelist-source-range" value for the K8s ingress for the application. 

The "whitelist-source-range" annotation just acts as a simple ACL to explicitly allow traffic from specific IPs. For example, we set it to "0.0.0.0/0" for production as we don't want to restrict traffic to the application from the Internet. For development, we usually lock it down to just the Boley IP.